plugins/gatsby-plugin-generate-doc-json/utils/handlers.js

```diff
@@ -13,23 +17,50 @@ const getAllAttributes = (node) =>
       }, {})
     : {};
 
+const getSrcUrl = (fileRelativePath, url) =>
+  fileRelativePath
+    .replace('.mdx', '')
+    .replace('src/content/', '')
+    .split('/')
+    .slice(0, -1)
+    .join('/')
+    .concat(url.replace('./', '/'));
+
 module.exports = {
+  paragraph: (h, node, imageHashMap, fileRelativePath) => {
+    if (hasOnlyChild('image', node)) {
+      const srcUrl = getSrcUrl(fileRelativePath, node.url);
+      return h(
+        node,
+        'p',
+        {
+          className: 'article-image',
+        },
+        [
+          h(node, 'img', {
+            src: imageHashMap[srcUrl] || node.url,
+            alt: node.alt,
+            className: 'article-image',
+          }),
+        ]
+      );
+    }
+    return all(h, node);
+  },
   image: (h, node, imageHashMap, fileRelativePath) => {
-    const srcUrl = fileRelativePath
-      .replace('.mdx', '')
-      .replace('src/content/', '')
-      .split('/')
-      .slice(0, -1)
-      .join('/')
-      .concat(node.url.replace('./', '/'));
+    const srcUrl = getSrcUrl(fileRelativePath, node.url);
     return h(
       node,
-      'img',
+      'div',
```

Comment:
Since this is outputting to a `div`, how does this work when the image is used inline with text? 

Here are a couple pages that show this:

* https://docs-preview.newrelic.com/docs/infrastructure/install-infrastructure-agent/update-or-uninstall/uninstall-infrastructure-agent
* https://docs-preview.newrelic.com/docs/infrastructure/install-infrastructure-agent/get-started/install-infrastructure-agent